firmware/Capability.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _CAPABILITY_H
+#define _CAPABILITY_H
+#include <Servo.h>
+#include <AccelStepper.h>
+
+class Capability {
+    public:
+        Capability();
+        virtual char type();
```

Comment:
Ovu metodu deklarisati kao apstraktnu:
```suggestion
        virtual char type() = 0;
```
Ovo će celu klasu napraviti apstraktnom i sprečiti pravljenje njenih objekata.